lib/common/client/middleware/response/facility_parser.rb

```diff
@@ -63,6 +63,12 @@ def make_complex_mappings
               end
             end
 
+            def map_classification
+              mapper = get_mapper('classification')
+              src_value = @src_facility['attributes'][@mapping['classification']]
+              mapper.nil? ? src_value : mapper.call(src_value, @src_facility)
```

Comment:
Maybe we could check the facility type here instead? For example, if the facility type is vha, then call the `classification_mapping` method

Reply:
> Maybe we could check the facility type here instead? For example, if the facility type is vha, then call the `classification_mapping` method

I was trying to keep it generic so there were no facility type checks in the parser but I can add it to simplify that logic